vms/platformvm/state/camino.go

```diff
@@ -83,6 +98,16 @@ type caminoState struct {
 	depositOffers     map[ids.ID]*DepositOffer
 	depositOffersList linkeddb.LinkedDB
 	depositOffersDB   database.Database
+
+	// Proposals
+	proposals    map[ids.ID]*Proposal
+	proposalList linkeddb.LinkedDB
+	proposalsDB  database.Database
+
+	// Vote
+	votes    map[ids.ID]*Vote
```

Comment:
What is the reason for separating votes and proposals?

Reply:
Mental clarity, i changed it cuz this is just a caching structure